### ISSUE TITLE
Error: Create custom endpoint with "npx strapi generate"

### ISSUE THREAD

**Issue by trofimenk0** [strapi/strapi#22489]
# Error with "npx strapi generate" command
## System information:
* Windows 11
* Node v20.18.1
* Strapi v5.5.1

## Algorithm of actions:
* Install Strapi v5.5.1
* Run the CLI command `npx strapi generate`
* Try to create a Custom API Endpoint.

```bash
PS E:\Work\WebDev\Strapi\Clear Strapi\pure-strapi> npx strapi generate
? Strapi Generators api - Generate a basic API
? API name content-summary
? Is this API for a plugin? No
✔  ++ \api\content-summary\routes\content-summary.ts
✔  ++ \api\content-summary\controllers\content-summary.ts
✔  ++ \api\content-summary\services\content-summary.ts
```
All seems good, but no folder and files were created in scr/api folder.

## Expected behavior:
Files will be created and can be edited

## Console output on attempt to generate again
```bash
PS E:\Work\WebDev\Strapi\Clear Strapi\pure-strapi> npx strapi generate
? Strapi Generators api - Generate a basic API
? API name content-summary
? Is this API for a plugin? No
✖  ++ File already exists
 -> E:\Work\WebDev\Strapi\Clear Strapi\pure-strapi\node_modules\@strapi\generators\dist\api\content-summary\routes\content-summary.ts
✖  ++ api/{{ id }}/controllers/{{ id }}.ts Aborted due to previous action failure
✖  ++ api/{{ id }}/services/{{ id }}.ts Aborted due to previous action failure
```
 But still no folder and no files.
 
 No new options for controlling access to the new endpoint have appeared in the admin panel. It seems as if the endpoint does not exist.
 
 A year ago it worked but not now.

**Comment by iamandrewluca** [comment]
The same issue, and it seems, is not only regarding API

```
? Strapi Generators api - Generate a basic API
? API name admin
? Is this API for a plugin? Yes
[ERROR] Couldn't find a "plugins" directory
```

```
node -v
v22.12.0
npm -v
10.9.1
uname
Darwin
npm ls @strapi/strapi --depth 0
├── @strapi/strapi@5.6.0
```

**Comment by derrickmehaffy** [comment]
fixed in: https://github.com/strapi/strapi/pull/22443

Will be released in v4.6.1

**Comment by warlord987** [comment]
> fixed in: #22443
> 
> Will be released in v4.6.1

v4.6.1 or v5.6.1?
